File: Estruturas de Dados/Vetor.c
#include <stdio.h>

int main() {
    
    int maior, i, num, pos, vetor[8];

    for (i = 0; i < 8; i++) {
        printf("Informe número na posição %d: ", i); scanf("%d", &num);
        vetor[i] = num;
    }

    maior = vetor[0]; pos = 0;
    for (i = 0; i < 8; i++) {

        if (vetor[i] > maior) {
            maior = vetor[i];
            pos = i;
        }
    }

    printf("O maior valor é %d, na posição %d\n", maior, pos);

    return 0;
}

File: Estruturas de Dados/Vetor.py
vetor = [4, 1, 3, 7, 13, 6, -5, 9]
elemento = vetor[0]
i = 0

for valor in vetor:

    if valor > elemento:
        elemento = valor
        pos = i

    i = i + 1

print(f"maior valor é {elemento} na posição {pos}")